www/src/models/transformation/TransformationStore.ts

```diff
@@ -0,0 +1,38 @@
+import { SimpleCRUDStore } from "gena-app";
+import { SERVER } from "../../env";
+import {TransformationTable} from "./TransformationResult";
+import axios from "axios";
+
+
+export interface Transformation {
+  id: number;
+};
+
+export class TransformationStore extends SimpleCRUDStore<number, Transformation> {
+  constructor() {
+    super(`${SERVER}/api/transformation`, undefined, false);
+  }
+ 
+  public filterErrorMessage(errorMessage: string) : string {
+    return errorMessage.split(':').splice(1).join(':').trim();
+  }
+  
+  async postData(id : number, payload: TPayload) : Promise<TransformationTable[]|undefined> {
```

Comment:
the name is too generic, if the function is for testing transformation, say that.

Reply:
renamed it as postTransformationTest .